### PR TITLE
Flow label

### PR DESCRIPTION
**Pull request by kvermeul**

### CODE REVIEW

File: src/probe.cpp [hunk 1]
```diff
@@ -36,29 +36,33 @@ Probe Probe::from_csv(const std::string &line) {
...
         probe.wait_us = Checked::stou32(token);
         break;
+
       default:
         break;
     }
     index++;
   }
-  if (index < 5 || index > 6) {
+  if (index < 6 || index > 7) {
```
Comment (maxmouchet) [inline src/probe.cpp:50]: Can we make this `index < 5 || index > 7` so that specifying the flow label is not mandatory?

Comment (kvermeul) [reply src/probe.cpp:50]: How do we know then which optional field to select between the flow_label and the wait_us?

Comment (maxmouchet) [reply src/probe.cpp:50]: If the user wants to set `wait_us` he can just set `flow_label` to 0.

Comment (maxmouchet) [reply src/probe.cpp:50]: So `flow_label` is always 6th and `wait_us` 7th.

Comment (kvermeul) [reply src/probe.cpp:50]: OK, sounds good

File: python/pycaracal.cpp [hunk 2]
```diff
@@ -131,6 +135,7 @@ PYBIND11_MODULE(_pycaracal, m) {
       .def_readwrite("reply_mpls_labels", &Reply::reply_mpls_labels)
       .def_readwrite("probe_dst_addr", &Reply::probe_dst_addr)
       .def_readwrite("probe_id", &Reply::probe_id)
+      .def_readwrite("probe_id", &Reply::probe_flow_label)
```
Comment (maxmouchet) [inline python/pycaracal.cpp:138]: `probe_flow_label`

**Review by maxmouchet — APPROVED** [review]

**Pull request merged**